jproject-fr/easy-eat
# #K591: Created the shipping address select page for poc+gitignore upadted

Added a new page for poc where user can select an adress for shipping. When the adress is selected the bg change. Also created a component for checkout breadcrumb. Finally updated the .gitignore to ignore the local front dockerfile.
To Test:
 - Go to /poc/shippingaddress
 - You should have 
   
![image](https://github.com/jproject-fr/easy-eat/assets/117679829/b862f69e-e67c-4394-be05-0030f3488c4c)
- When you click on an address the background should change and the selected address will have a purple background.
![image](https://github.com/jproject-fr/easy-eat/assets/117679829/3506a549-bf6c-44d3-91a7-fbb2ee6b4b68)
![image](https://github.com/jproject-fr/easy-eat/assets/117679829/ee6cc129-9f9a-487d-b7af-4d9bac6ca7cb)
![image](https://github.com/jproject-fr/easy-eat/assets/117679829/a1a61c45-4177-476f-a20c-23606048d024)
